ibridges/util.py

```diff
@@ -76,3 +82,56 @@ def obj_replicas(obj: irods.data_object.iRODSDataObject) -> list[tuple[int, str,
                  r.size, repl_states.get(r.status, r.status)) for r in obj.replicas]
 
     return replicas
+
+def get_environment_providers() -> list:
+    """Get a list of all environment template providers.
+
+    Returns
+    -------
+        The list that contains the providers.
+
+    """
+    return [entry.load() for entry in entry_points(group="ibridges_server_template")]
```

Comment:
So all python packages contain "ibridges_server_tamplate" will be found and can be employed? Neat!!!